### PR TITLE
Properly escape attribute values

### PR DESCRIPTION
**Pull request by lamplightdev**
Fixes #25

### CODE REVIEW

File: lib/codegen.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ function * walkElement(node, state) {
       yield htmlValue(` ${name}`);
     } else {
       yield htmlValue(` ${name}="`);
-      yield * multiInterpolation(value, state);
+      yield * multiInterpolation(value.replace(/"/g, '&quot;'), state);
```
[the remark this answers is not in the document]
Comment (matthewp) [reply lib/codegen.js:52]: That part is safe.

**Comment by matthewp** [comment]
Can you create a function called `escapeAttributeValue` or something like that which does the replace? Otherwise it looks good to me.

**Comment by lamplightdev** [comment]
Great, I'll get this done later today

**Comment by matthewp** [comment]
Perfect!

**Comment by matthewp** [comment]
This was released in 1.2.4

**Pull request merged**